pkg/goGit/goGit.go

```diff
@@ -11,14 +13,18 @@ import (
 	logr "github.com/sirupsen/logrus"
 )
 
-func UpdateLocalCopies(names []string, config *config.Config) {
+func UpdateLocalCopies(names []string, config *config.Config, user *config.User) {
 	for i := 0; i < len(names); i++ {
-		// Open the repository at the give path
-		r, err := git.PlainOpen(config.OutputPath + names[i])
+		r, err := git.PlainOpen(filepath.Join(config.OutputPath, names[i]))
 		if err != nil {
 			if err == git.ErrRepositoryNotExists {
 				logr.Errorf("[Git] Couldn't find a local copy of Repository %v", names[i])
-				Clone(names[i], config)
+
+				if user != nil {
+					Clone(names[i], config, user)
+				} else {
+					Clone(names[i], config, nil)
+				}
```

Comment:
Unnecessary If statement the third parameter 'User' is passed anyway in the called function.

Reply:
fixed in commit 0a1f9ae